index.js

```diff
@@ -45,15 +45,47 @@ module.exports = function (config, deps) {
   config.metricsSource = config.metricsSource.replace(/-/g, ' ');
 
   var common = require('./lib/common.js')(config, deps);
-  var confirm = require('./confirm.js')( common, {superagent:superagent, findProfile: findProfile});
+  var confirm = require('./confirm.js')( common, {superagent:superagent, findProfile: function (userId, cb) {
```

Comment:
Oh, wait, now I remember. That didn't work because the `findProfile` was in the return, rather than defined within the scope where `confirm` could use it. But what I can do is pull it out of the return, so it's within scope, and then have the return reference the scoped function.